gulpfile.js

```diff
@@ -130,7 +130,6 @@ function amdify(source, subDependencyMapping) {
   let requirePath;
 
   source = source.replace(/\r\n/g, "\n");
-  source = source.replace(/\b(let|const)\b/g, "var");
```

Comment:
No reason to use `var` anymore especially now that CesiumJS has dropped support for IE11.

Reply:
does this also apply to CesiumJS source code?

Reply:
That's for us to decide. The `GltfPipeline` folder in CesiumJS is very self-contained so it's a good testing ground for things like this.